cartoframes/data/observatory/enrichment/enrichment_service.py

```diff
@@ -20,7 +20,8 @@ class EnrichmentService(object):
     """Base class for the Enrichment utility with commons auxiliary methods"""
 
     def __init__(self, credentials=None):
-        self.bq_user_dataset = BQUserDataset(credentials=credentials or get_default_credentials())
+        auth_client = _create_auth_client(credentials or get_default_credentials())
+        self.bq_user_dataset = DODataset(auth_client=auth_client)
```

Comment:
we have to remove those `bq_` references

Reply:
Ouch!